rusk/src/lib/node/vm/config.rs

```diff
@@ -115,8 +119,8 @@ impl Config {
 
     /// Create a new `Config` with the given parameters.
     pub fn to_execution_config(&self, block_height: u64) -> ExecutionConfig {
-        let with_public_sender = self
-            .feature("ABI_PUBLIC_SENDER")
+        let with_public_sender: bool = self
+            .feature(feature::FEATURE_ABI_PUBLIC_SENDER)
```

Comment:
what does it have to do with finalization order?

Reply:
Additionally, since that String is now used in 2 places, I've created a const